adit-client/adit_client/client.py

```diff
@@ -67,6 +69,136 @@ def retrieve_study(
             additional_params=additional_params,
         )
 
+    def retrieve_nifti_study(self, ae_title: str, study_uid: str) -> list[tuple[str, BytesIO]]:
```

Comment:
Besides the `retrieve_` methods, there are also `iter_` methods for study and series. These are even the most frequently used ones, as files can be saved while other files are still downloading, and also not everything is accumulating in memory. These are still missing for Nifti in this PR.